src/plonemeeting/portal/core/content/institution.py

```
@@ -589,26 +635,43 @@ def fetch_delib_representatives(self):
+        if (
+            self.plonemeeting_url
+            and self.meeting_config_id
+            and self.username
+            and self.password
+        ):
             if url:
-                logger.info("Fetching {} for {} [Start]".format(attr_name, self.title))
+                logger.info(f"Fetching {attr_name} for {self.title} [Start]")
                 try:
-                    response = requests.get(url, auth=(self.username, self.password), headers=API_HEADERS)
+                    response = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)